scripts/zones/Buburimu_Peninsula/Zone.lua

```diff
@@ -4,60 +4,81 @@
 --
 -----------------------------------
 local ID = require("scripts/zones/Buburimu_Peninsula/IDs")
-require("scripts/globals/icanheararainbow");
-require("scripts/globals/chocobo_digging");
-require("scripts/globals/conquest");
+require("scripts/globals/icanheararainbow")
+require("scripts/globals/chocobo_digging")
+require("scripts/globals/conquest")
 require("scripts/globals/helm")
-require("scripts/globals/zone");
+require("scripts/globals/zone")
 -----------------------------------
 
 function onChocoboDig(player, precheck)
     return tpz.chocoboDig.start(player, precheck)
-end;
+end
 
 function onInitialize(zone)
+    local hour = VanadielHour()
+    
+    if hour >= 6 and hour < 16 then
+        DisallowRespawn(ID.mob.BACKOO, false)
```

Comment:
Should just need to change this to `GetMobByID(ID.mob.BACKOO):setRespawnTime(1)`